src/main/scala/diplomaticobjectmodel/model/ISASpecifications.scala

```diff
@@ -37,4 +40,12 @@ object ISAExtensions {
   )
 
   def specVersion(extension: OMExtensionType, version: String): OMSpecification = OMSpecification(specifications(extension), version)
+
+  def customExtensions(coreParams: CoreParams): List[OMCustomExtensionSpecification] = {
+    val cflush = coreParams.haveCFlush.option(Xsifivecflushdlone("", ""))
+    List[Option[OMCustomExtensionSpecification]](cflush).flatten
```

Comment:
`if(coreParams.haveCFlush) List(Xsifivecflushdlone("", "") else Nil`

Reply:
There can be more than one custom extension.